src/main/java/org/traccar/forward/PositionForwarderWialon.java

```diff
@@ -108,4 +111,28 @@ public void forward(PositionData positionData, ResultHandler resultHandler) {
         }
     }
 
+    public static String convertJsonToWialonParams(Map<String, Object> attributes) {
```

Comment:
Maybe just call this `formatAttributes`. It definitely shouldn't have JSON in the name because there's not JSON. And also it doesn't make sense to use Wialon in the name because the whole class is for Wialon only.